src/app/shell/personal-cabinet/shared-cabinet/applications/applications.component.scss

```diff
@@ -15,6 +15,10 @@
   margin-bottom: 1rem;
 }
 
+::ng-deep .mat-form-field-infix {
```

Comment:
This will change the styles for all elements with the `.mat-form-field-infix` class in the entire application with ng-deep.
Add an additional class that will only match the linked form fields.